packages/nodes-base/nodes/ConvertKit/ConvertKit.node.ts

```diff
@@ -0,0 +1,287 @@
+			},
+			//--------------------
+			// Field Description
+			//--------------------
+			...fieldOperations,
+			...fieldFields,
+			//--------------------
+			// FormDescription
+			//--------------------
+			...formOperations,
+			...formFields,
+			//--------------------
+			// Sequence Description
+			//--------------------
+			...sequenceOperations,
+			...sequenceFields,
+			//--------------------
+			// Tag Description
+			//--------------------
+			...tagOperations,
+			...tagFields,
+		],
+	};
+
+	async execute(this: IExecuteFunctions): Promise<INodeExecutionData[][]> {
+		const items = this.getInputData();
+		const returnData: IDataObject[] = [];
+
+		let method = '';
+		let endpoint = '';
+		const qs: IDataObject = {};
+		let responseData;
+
+		const resource = this.getNodeParameter('resource', 0) as string;
+		const operation = this.getNodeParameter('operation', 0) as string;
+
+		const fullOperation = `${resource}/${operation}`;
+
+		for (let i = 0; i < items.length; i++) {
+			//--------------------
+			// Field Operations
+			//--------------------
+			if(resource === 'field') {
+				//---------
+				// Update
+				//---------
+				if(operation === 'update') {
+					qs.label = this.getNodeParameter('label', i) as string;
+
+					const id = this.getNodeParameter('id', i) as string;
+
+					method = 'PUT';
+					endpoint = `/custom_fields/${id}`;
+				//---------
+				// Get All
+				//---------
+				} else if(operation === 'getAll') {
+					method = 'GET';
+					endpoint = '/custom_fields';
+				//---------
+				// Create
+				//---------
+				} else if(operation === 'create') {
+					qs.label = this.getNodeParameter('label', i) as string;
+
+					method = 'POST';
+					endpoint = '/custom_fields';
+				//---------
+				// Delete
+				//---------
+				} else if(operation === 'delete') {
+					const id = this.getNodeParameter('id', i) as string;
+
+					method = 'DELETE';
+					endpoint = `/custom_fields/${id}`;
+				}
+			//--------------------------------------------
+			// Form, Sequence, and Tag Operations
+			//--------------------------------------------
+			} else if(['form', 'sequence', 'tag'].includes(resource)) {
+				//-----------------
+				// Add Subscriber
+				//-----------------
+				if(operation === 'addSubscriber') {
+					qs.email= this.getNodeParameter('email', i) as string;
+					const id = this.getNodeParameter('id', i);
+
+					const additionalParams = this.getNodeParameter('additionalFields', 0) as IDataObject;
+
+					if(additionalParams.firstName) {
+						qs.first_name = additionalParams.firstName;
+					}
+
+					if(additionalParams.fields !== undefined) {
+						const fields = {} as IDataObject;
+
+						// @ts-ignore
+						for(let j = 0; j < additionalParams.fields.field.length; j++) {
+							// @ts-ignore
+							const key = additionalParams.fields.field[j].key as string;
+							// @ts-ignore
+							const value = additionalParams.fields.field[j].value;
```

Comment:
Is it possible to use the optional chaining operator to do away with the TypeScript exceptions?